docs/change-log.md

```diff
@@ -2,6 +2,14 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update April 21st, 2023
+
+- **New:** Introduced [Versioning of Shared Values and overrides](https://docs.humanitec.com/using-humanitec/work-with-shared-values-and-secrets/versioning-of-values-and-secrets).
+- **Improved:** The [deployment diff](https://docs.humanitec.com/using-humanitec/work-with-environments/deploy-to-environments/diff-deployments) UI has been revamped, making it easier for developers to compare app states and troubleshoot issues. Check out [the release announcement on our blog](https://humanitec.com/blog/spot-the-difference-introducing-humanitecs-improved-deployment-diff).
+- **Improved:** Added support for viewing error messages of past failed deployments.
+- **Improved:** It is now possible to view the active resources of Humanitec's default resource definitions in the Usage tab of the [Resource Definition Details screen](https://docs.humanitec.com/reference/user-interface/resource-management#resource-definition-details)
+- **Improved:** Deployment diff now shows changes even if the application hasn't been deployed.
+
```

Comment:
```suggestion
- **Improved:** The design of [private](https://docs.humanitec.com/using-humanitec/work-with-workloads/manage-resource-dependencies) and [shared resources](https://docs.humanitec.com/using-humanitec/work-with-apps/shared-resources) of running deployments. A direct link to the connected [Resource Definition](https://docs.humanitec.com/guides/orchestrate-infrastructure/dynamic-vs-static-resource-definitions) has been added for [Organization Administrators](https://docs.humanitec.com/guides/manage-users-permissions/role-based-access-control#organization-roles). 
```

Reply:
This can be added once we updated the docs https://github.com/humanitec/docs-site-gitbook/pull/67